Cargo.toml

```diff
@@ -0,0 +1,56 @@
+################################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+
+[package]
+author = "Eclipse Foundation uProtocol project"
+documentation = "https://github.com/eclipse-uprotocol/up-spec#readme"
+edition = "2021"
+exclude = [
+    "tests/*",
+    "tools/*",
+    ".devcontainer/*",
+    ".vscode/*",
+    ".github/*",
+    ".gitignore",
+]
+keywords = ["uProtocol", "SDK"]
+license = "Apache-2.0"
+name = "up-client-mqtt5-rust"
+readme = "README.md"
+repository = "https://github.com/eclipse-uprotocol/up-client-mqtt5-rust"
+rust-version = "1.72"
+version = "0.1.0"
+
+[dependencies]
+async-trait = { version = "0.1" }
+bytes = { version = "1.5" }
+chrono = { version = "0.4" }
+cloudevents-sdk = { version = "0.7" }
+mediatype = "0.19"
+paho-mqtt = { version = "0.12.1" }
+protobuf = { version = "3.3" }
+rand = { version = "0.8" }
+regex = { version = "1.10" }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0" }
+up-rust = { git = "https://github.com/eclipse-uprotocol/up-rust", version = "0.1.5" }
+url = { version = "2.5" }
+uuid = { version = "1.7", features = ["v8"] }
+
+[build-dependencies]
+protobuf-codegen = { version = "3.3" }
```

Comment:
Perhaps these are not needed here? They're used in `up-rust` in order to download and generate Rust code for the .protos.

Reply:
Good call, removed. I ended up removing some more dependencies and these can be readded if needed in the future.